staging/src/k8s.io/api/storage/v1/types.go

```diff
@@ -361,12 +361,15 @@ type CSIDriverSpec struct {
 	// most one token is empty string. To receive a new token after expiry,
 	// RequiresRepublish can be used to trigger NodePublishVolume periodically.
 	//
-	// This is an alpha feature and only available when the
+	// This is a beta feature and only available when the
 	// CSIServiceAccountToken feature is enabled.
 	//
 	// +optional
-	// +listType=atomic
-	TokenRequests []TokenRequest `json:"tokenRequests,omitempty" protobuf:"bytes,6,opt,name=tokenRequests"`
+	// +patchMergeKey=audience
+	// +patchStrategy=merge
```

Comment:
@apelisse can you confirm that changing patchMergeKey / patchStrategy is not compatible with client-side apply?